packages/functional-tests/tests/settings/recoveryKey.spec.ts

```diff
@@ -18,49 +15,37 @@ test.describe('severity-1 #smoke', () => {
         // Generating and consuming recovery keys is a slow process
         // Mail delivery can also be slow
         test.slow();
-
-        await settings.goto();
-        let status = await settings.recoveryKey.statusText();
-        expect(status).toEqual('Not Set');
-        await settings.recoveryKey.clickCreate();
-        // View 1/4 info
-        await recoveryKey.clickStart();
-        // View 2/4 confirm password and generate key
-        await recoveryKey.setPassword(credentials.password);
-        await recoveryKey.submit();
-
-        // View 3/4 key download
-        // Store key to be used later
-        key = await recoveryKey.getKey();
-        await recoveryKey.clickNext();
-
-        // View 4/4 hint
-        // store hint to be used later
-        hint = 'secret key location';
-        await recoveryKey.setHint(hint);
-        await recoveryKey.clickFinish();
-
-        // Verify status as 'enabled'
-        status = await settings.recoveryKey.statusText();
-        expect(status).toEqual('Enabled');
       }
     );
 
     test('can copy recovery key', async ({
       credentials,
       pages: { recoveryKey, settings },
     }) => {
+      await settings.goto();
+
+      await expect(settings.settingsHeading).toBeVisible();
+      await expect(settings.accountRecoveryKeyStatus).toHaveText('Not Set');
+
       // Create new recovery key
-      await settings.recoveryKey.clickCreate();
+      await settings.accountRecoveryKeyCreateButton.click();
```

Comment:
I think I understand what you mean here, that steps 1-3 out of 4 are repeated more frequently. I'm not keen to have the `fillOutRecoveryKeyForms` method support copy and download, I think this would add conditional complexity that wouldn't be best practice. The second idea of extracting the 'steps up to clicking `createKeyButton`' I like much better. 

I think the copy and download scenarios are in the `settings` tests right? This is still left to be audited in [FXA-8961](https://mozilla-hub.atlassian.net/browse/FXA-8961) If you are cool with it, I can pick that one up next and look at this reuse problem as part of that scope?

Reply:
SGTM!